CHANGELOG.md

```diff
@@ -87,6 +87,7 @@
 ### 🚀 New components 🚀
 
 - `sigv4authextension`: Enable component (#8518)
+- `prometheusexporter`: Prometheus fails to generate logs when prometheus exporter produced a check exception occurs. (#8949)
```

Comment:
This should be under "## Unreleased", "Bug Fixes". So around line 43

Reply:
I have completed the merger